modules/apps/document-library-opener/document-library-opener-google-drive-web/src/main/java/com/liferay/document/library/opener/google/drive/web/internal/DLOpenerGoogleDriveManager.java

```diff
@@ -208,6 +228,31 @@ public boolean isGoogleDriveFile(FileEntry fileEntry) {
 		);
 	}
 
+	public boolean isGoogleDriveFileExisting(long userId, FileEntry fileEntry) {
```

Comment:
Can you rename this to `hasGoogleDriveFile`?

Reply:
Thanks for your feedback. I update the code.